test/testPay.js

```diff
@@ -0,0 +1,125 @@
+/** Scenario: Alice pays Bob 10 REV.
+
+Alice has 100 REV at genesis. All she knows about Bob is his public key.
+
+ISSUE: should be: all she knows about Bob is his eth address.
+
+*/
+/* global require */
+// @flow
+
+const { GPrivate } = require('../protobuf/RhoTypes.js');
+const {
+  RNode, b2h, h2b, RHOCore, makeProxy, sendCall, keyPair, blake2b256Hash,
+} = require('..');
+const { link } = require('./assets');
+const { loadRhoModule, unforgeableWithId } = require('./loading');
+
+const defaultPayment = { from: '0x1', nonce: 0, phloPrice: 1, phloLimit: 100000 };
+const user = h2b('d72d0a7c0c9378b4874efbf871ae8089dd81f2ed3c54159fffeaba6e6fca4236'); // arbitrary
```

Comment:
So what is `user` supposed to actually mean? Is it Alice? Bob (probably not)? The validator?

Reply:
I don't know.

rnode doesn't check any signatures against it yet.